### DISCUSSION TITLE
<title>I have no more access to my old email bc its a school email so i cant enter my old account

### DISCUSSION THREAD

**Discussion by salva311** [community/community#126969] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

So i had an email addres where i had linked my old github account and i finished school and that email is no longer available so i cant enter my github account because it needs a passcode

**Comment by PythonScratcher** [comment]
you can try contact the school to temporarily give you the email back

**Reply by salva311** [reply]
yeah i did it before but they have no acces to my account because i am no longer a student there

**Comment by ebndev** [comment]
Hi @salva311,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
